orcestra_book/data_policy.md

```diff
@@ -0,0 +1,13 @@
+# Data Policy
+
+ORCESTRA intends to follow the open data policy established for NARVAL. This means that we do not intend to police the use of, or access to, the data. Instead we rely on the good will and scientific integrity of those interested in using the data to respect the efforts of those whose were responsible for its collection by adhering to the two guidelines below:
+
+1. No redistribution: Groups who are using the data are asked not to redistribute this data, but rather refer those interested in accessing the data to the original source, and these guidelines for the data use.
```

Comment:
This point may be in conflict with our planned use of IPFS, or more general with the idea of making data available in new and useful ways.

IPFS by design would make data available from any accessible copy (i.e. if there's a copy on-site at the field campaign, users can access it from there and if another copy is available elsewhere, like at DKRZ, users might access it from there as well). If we are using IPFS, people might even inadvertently redistribute the data, as that's the default behavior once you access the data.

Likewise (and independent of IPFS), if we should figure out that e.g. having a copy of the dataset in a better accessible form (like a single-dataset zarr instead of multi-dataset netCDF), this might also require redistributing the data.

Maybe we could find a formulation which emphasizes the importance to get in contact with the original authors of the data, especially when using the data, but which still allows improvements on the data presentation for the project as a whole.

Reply:
I see... so, how about the following formulation then:

"Groups using the data are encouraged to get in touch with the original authors to discuss their use of the data, particularly when making enhancements to data presentation or accessibility. While maintaining data integrity and traceability is important, the potential for new distribution methods such as through IPFS shall not be undermined. Therefore, we ask users to refer others interested in accessing the data to the original source whenever possible, ensuring proper attribution and encouraging direct communication with the original authors.
"  (?)

Reply:
That's quite a lot of text and contains lots of technical details which maybe not everyone will understand quickly. I'd see the data policy as a rather high level document which should communicate the goals, not so much the methods. Could we go for something like

```suggestion
1. Be kind and cooperative: When using the data, you should get in touch with and refer to the original source, and these guidelines for the data use.
```

Reply:
I'm fine with that. But maybe @bjorn-stevens have some suggestions, too?